pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 python = "^3.12"
 datasets = "*"
 pandas = "*"
-litellm = "*"
+litellm = { git = "https://github.com/xingyaoww/litellm.git", branch = "xw/fix-fn-call" }
```

Comment:
It'd be nice if we didn't have to do this, but maybe we can do it for now and open an issue to remember to revert and share it with the litellm folks?